### PR TITLE
Add template option to polarion report

### PR DESCRIPTION
**Pull request by KwisatzHaderach**
Add show_default to utils.field
Fix polarion report hostname/arch with use-facts property names

### CODE REVIEW

File: tmt/utils.py [hunk 1]
```diff
@@ -4839,6 +4839,7 @@ def field(
         envvar: Optional[str] = None,
         deprecated: Optional['tmt.options.Deprecated'] = None,
         help: Optional[str] = None,
+        show_default: Optional[str] = None,
```
Comment (happz) [inline tmt/utils.py:4842]: Isn’t it rather a `bool`, without `Optional`?

Comment (KwisatzHaderach) [reply tmt/utils.py:4842]: yeah, was thinking where the problem is and then rather went for lunch 😆

**Review by thrix — APPROVED** [review]
LGTM, no comments, besides 2 MRs for different changes would be appreciated next time.

**Review by janhavlin — APPROVED** [review]

**Review by psss — APPROVED** [review]
Looks good, thanks! Rebased on the latest `main`, will merge once tests are green.

**Pull request merged**